lms/static/sass/base/_base.scss

```diff
@@ -183,12 +183,9 @@ span.edx {
 }
 
 .loading-animation {
```

Comment:
Found it. Updating the SB now: https://alisan617.sandbox.edx.org/template/student_account/finish_auth.html